pkg/controller/backup_controller.go

```diff
@@ -670,6 +670,16 @@ func (c *backupController) runBackup(backup *pkgbackup.Request) error {
 		if err != nil {
 			backupLog.Errorf("fail to wait VolumeSnapshot change to Ready: %s", err.Error())
 		}
+
```

Comment:
We should move the code to list volumesnapshots and assigning the list to backup request  into `c.checkVolumeSnapshotReadyToUse`

 btw rename it to `waitVolumeSnapshotReadyToUse` would be better.